appdev/apparmor.rst

```diff
@@ -0,0 +1,295 @@
+
+Usage: reserved
+
+
+keep-display-on
+---------------
+
+Description: Can request keeping the screen on
+
+Usage: common
+
+
+location
+--------
+
+Description: Can access Location
+
+Usage: common
+
+Allows an app to request access to the device's current location.
+
+
+microphone
+----------
+
+Description: Can access the microphone
+
+Usage: common
+
+
+music_files
+-----------
+
+Description: Can read and write to music files. This policy group is reserved for certain applications, such as music players. Developers should typically use the content_exchange policy group and API to access music files instead.
+
+Usage: reserved
+
+The music_files policy group allows an app to read or write to the Music directories in the user's home folder or on external media.
+
+
+music_files_read
+----------------
+
+Description: Can read all music files. This policy group is reserved for certain applications, such as music players. Developers should typically use the content_exchange policy group and API to access music files instead.
+
+Usage: reserved
+
+The music_files policy group allows an app to read the Music directories in the user's home folder or on external media.
+
+
+networking
+----------
+
+Description: Can access the network
+
+Usage: common
+
+The networking policy group allows an app to contact network devices and use the `download manager`_.
+
+
+picture_files
+-------------
+
+Description: Can read and write to picture files. This policy group is reserved for certain applications, such as gallery applications. Developers should typically use the content_exchange policy group and API to access picture files instead.
+
+Usage: reserved
+
+The music_files policy group allows an app to read and write to the Pictures directories in the user's home folder or on external media.
+
+
+picture_files_read
+------------------
+
+Description: Can read all picture files. This policy group is reserved for certain applications, such as gallery applications. Developers should typically use the content_exchange policy group and API to access picture files instead.
+
+Usage: reserved
+
+The music_files policy group allows an app to read the Pictures directories in the user's home folder or on external media.
+
+
+push-notification-client
+------------------------
+
+Description: Can use push notifications as a client
+
+Usage: common
+
+
+sensors
+-------
+
+Description: Can access the sensors
+
+Usage: common
+
+Allows apps to access `device sensors`_
+
+
+usermetrics
+-----------
+
+Description: Can use UserMetrics to update the InfoGraphic
+
+Usage: common
+
+Allows an app to write metrics to the UserMetrics service so they can be displayed on the InfoGraphic.
+
+
+video
+-----
+
+Description: Can play video (allows playing remote content via media-hub)
+
+Usage: common
+
+
+video_files
+-------------
+
```

Comment:
Extra dashes

Reply:
Shouldn't matter, the rst specification says it just needs to be at least as long as the text above. But i guess for the sake of continuity, it could be removed.

Reply:
It does matter, but my linter hasn't been working lately...